apisyouwonthate/apisyouwonthate.com
# Add AMA page for taking podcast questions/suggestions

Added a quick form to take suggestions.  I'll be hooking it up to Netlify forms.

To test:
go to /ama and fill out the form.  You should see a (fairly blank) thank you page on submit:

![image](https://user-images.githubusercontent.com/1844496/65984717-f6712f80-e445-11e9-822f-12ec5fe6fc94.png)


Note: there's a honeypot field that is hidden from view, which Netlify should use to filter out spam
